tests/gtest_dlt_user.cpp

```diff
@@ -3337,6 +3341,60 @@ TEST(t_dlt_user_log_write_sized_utf8_string, nullpointer)
     EXPECT_LE(DLT_RETURN_OK, dlt_unregister_app());
 }
 
+/*/////////////////////////////////////// */
+/* t_dlt_user_log_write_constant_utf8_string */
+TEST(t_dlt_user_log_write_constant_utf8_string, normal)
```

Comment:
I prefer to split the test case between normal case and non-verbose case.
Also please test nullptr case as well like others.

Reply:
OK